src/frontend/src/pages/SettingsPage/UserSettings/UserSettingsEdit.tsx

```diff
@@ -61,12 +62,13 @@ const UserSettingsEdit: React.FC<UserSettingsEditProps> = ({ currentSettings, on
           defaultValue={currentSettings.slackId}
           render={({ field: { onChange, value } }) => (
             <>
-              <Typography>
-                {'Slack Id '}
-                <Link href="https://www.workast.com/help/article/how-to-find-a-slack-user-id/">
-                  (How to find your Slack ID)
-                </Link>
-              </Typography>
+              <Grid container>
+                <Typography>{'Slack Id '}</Typography>
+                <ExternalLink
+                  link="https://www.workast.com/help/article/how-to-find-a-slack-user-id/"
+                  description="(How to find your Slack ID)"
+                ></ExternalLink>
```

Comment:
this can be one tag: <ExternalLink props/>; also the way it's being used here, I'm thinking this should be a div with a style prop set to {display: 'flex'} instead of the grid component to get the same effect, since grid isn't really being used as a grid here